odd-platform-ui/src/components/DataModelling/Relationships/RelationshipsSearchInput.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useCallback } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { SearchInput } from 'components/shared/elements';
+
+interface Props {
+  value?: string;
+}
+
+const RelationshipsSearchInput = ({ value }: Props) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const handleSearch = useCallback(
+    (v?: string) => {
+      searchParams.set('q', v ?? '');
+      setSearchParams(searchParams);
+    },
+    [searchParams]
+  );
+
+  return (
+    <SearchInput
+      id='relationships-search'
+      placeholder='Search relationships'
```

Comment:
Use a translation instead of this hardcoded string.

Reply:
I was thinking about translation. Due we don't support it anyway and it doesn't cover even 50% of UI. I think we should get rid off internationalization at all because we cannot support and it feels unnecessary right now. There are more in it's current structure  - it's easier to start doing it from the scratch then fix what we have.